src/AuthKeyValueUser.php

```diff
@@ -277,4 +258,14 @@ protected function getAuthKeyValueUserService()
     {
         return ServiceManager::getServiceManager()->get(AuthKeyValueUserService::SERVICE_ID);
     }
+
+    /**
+     * @return array
+     */
+    private function getKeyValueUserData() {
+        $login = current($this->getPropertyValues(GenerisRdf::PROPERTY_USER_LOGIN));
+        $userData = $this->getAuthKeyValueUserService()->getUserData($login);
+        $params = json_decode($userData[AuthKeyValueUserService::USER_PARAMETERS],true);
```

Comment:
is it possible that `AuthKeyValueUserService::USER_PARAMETERS`  might not exist in the array?

Reply:
fixed